packages/devtools_app/lib/src/extensions/extension_service.dart

```diff
@@ -11,27 +11,91 @@ import '../shared/primitives/auto_dispose.dart';
 
 class ExtensionService extends DisposableController
     with AutoDisposeControllerMixin {
+  /// All the DevTools extensions that are available for the connected
+  /// application, regardless of whether they have been enabled or disabled
+  /// by the user.
   ValueListenable<List<DevToolsExtensionConfig>> get availableExtensions =>
       _availableExtensions;
   final _availableExtensions = ValueNotifier<List<DevToolsExtensionConfig>>([]);
 
+  /// DevTools extensions that are visible in their own DevTools screen (i.e.
+  /// extensions that have not been manually disabled by the user).
+  ValueListenable<List<DevToolsExtensionConfig>> get visibleExtensions =>
+      _availableExtensions;
+  final _visibleExtensions = ValueNotifier<List<DevToolsExtensionConfig>>([]);
```

Comment:
Since we are not adding or removing values to the list, but rather resetting the value with a new list each time, it makes more sense to be a regular ValueNotifier. ListValueNotifier is best fit for places where we want to modify the list and still have listeners be notified. In this case, we are only ever assigning a new list to the notifier `_visibleExtensions.value = visible`. So using a ListValueNotifier would make us have to call `..clear()..addAll(visible)`

Reply:
Good points. Leaving it as-is SGTM.